.github/workflows/dependabot-update-all.yml

```diff
@@ -0,0 +1,38 @@
+name: Dependabot Update All Go Modules
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1.1.1
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+      # The following properties are now available:
+      #  - steps.metadata.outputs.dependency-names
+      #  - steps.metadata.outputs.dependency-type
+      #  - steps.metadata.outputs.update-type
+      - name: Update all Go modules
+        if: ${{ steps.metadata.outputs.package-ecosystem == 'gomod' }}
+        run: |
+          gh pr checkout "$PR_URL"
+          ./scripts/go-update-dep-all.sh ${{ steps.dependabot-metadata.outputs.updated-dependencies-json }}
```

Comment:
This is probably wrong, but I cannot find any docs about how looks `steps.dependabot-metadata.outputs.updated-dependencies-json` (https://github.com/dependabot/fetch-metadata).
I think we could print maybe the whole metatada and in a follow-up fix the action. Any thoughts?

Reply:
This doesn't feel right...

Reply:
Yes, let me fork the SDK actually and try there.

Reply:
Fixed